manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -44,6 +44,9 @@ spec:
         {{- else }}
         istio: pilot
         {{- end }}
+        {{- range $key, $val := .Values.pilot.pilotLabels }}
```

Comment:
rename `pilotLabels` to `labels` or `selectorLabels`.mSame for `operatorLabels`

Reply:
@jacob-delgado  I renamed it. Could you please review it one more time.